python/src/filesystem.rs

```diff
@@ -61,7 +61,7 @@ impl DeltaFileSystemHandler {
         let to_path = Path::from(dest);
         self.rt
             .block_on(self.inner.copy(&from_path, &to_path))
-            .map_err(PyDeltaTableError::from_object_store)?;
+            .map_err(PythonError::from)?;
```

Comment:
🤔 I wonder if we implemented:

```rust
impl<T: Into<PythonError>> From<T> for PyErr { ... }
```

could we remove the `map_err` entirely?

Reply:
Cool idea, and gave it a quick go, but the compiler is complaining that `Into` is not our trait, and neither is `PyErr`, so it won't allow it :(